lib/pypy.js

```diff
@@ -23,8 +23,8 @@ if (typeof module !== "undefined") {
 var debug = function(){};
 if (typeof console !== "undefined") {
   debug = console.log;
```

Comment:
I have this change locally as well, so I'll do that update when I merge from this PR.